### PR TITLE
Add CI and Docker

### PR DESCRIPTION
**Pull request by matt-**
- Add docker information and Github / Gitlab links
- Added gitlab and docker links

### CODE REVIEW

**Review by sip49 — CHANGES_REQUESTED** [review]
@matt- The GitHub link doesn't work for me? 404.

File: docs/open-pixee.md [hunk 1]
```diff
@@ -21,15 +21,12 @@ Learn more in the [README](https://github.com/pixee/pixee-cli#readme)
 Do you want to see Pixee at work in your repository? We've integrated the PixeeCLI into CI scripts for popular platforms.
 
 - [Bitbucket](https://github.com/pixee/pixee-cli/blob/main/ci/bitbucket/README.md)
-
-Coming soon!
-
-- Gitlab
-- Github
+- [GitLab](https://github.com/pixee/pixee-cli/blob/main/ci/gitlab/README.md)
+- [GitHub](https://github.com/pixee/ci-integrations/tree/main/docs/github.md)
```
Comment (dhafley) [inline docs/open-pixee.md:25]: 404

Comment (matt-) [reply docs/open-pixee.md:25]: These were waiting on their own PRs, and should be working now.

**Review by sip49 — APPROVED** [review]
lgtm, link works

**Pull request merged**